docs/developer/migrate/from-ethereum.md

```diff
@@ -23,8 +23,7 @@ For a general overview of the Celo network and architecture, see [the Celo Overv
 
 ## What is Celo's Relationship to Ethereum?
 
-Celo is a layer 1 protocol and blockchain platform, and the Celo Mainnet is entirely separate from the Ethereum network.
-While the Celo client originated as a fork of Ethereum Go language client, [go-ethereum](https://github.com/ethereum/go-ethereum) (or geth), it has several significant differences, including a proof-of-stake based PBFT consensus mechanism. All the cryptoassets on Celo have ERC-20 compliant interfaces, meaning that while they are not ERC-20 tokens on the Ethereum Mainnet, all familiar tooling and code that support ERC-20 tokens can be easily adapted for Celo assets, including the Celo Native Asset (CELO) and the Celo Dollar (cUSD).
+Celo is an Ethereum Layer 2 solution. While the Celo client originated as a fork of Ethereum Go language client, [go-ethereum](https://github.com/ethereum/go-ethereum) (or geth), it has several significant differences, including a proof-of-stake based PBFT consensus mechanism. All crypto assets on Celo have **ERC-20 compliant interfaces**, meaning that while they are not ERC-20 tokens on the Ethereum Mainnet, all familiar tooling and code that support ERC-20 tokens can be easily adapted for Celo assets, including the Celo Native Asset (CELO) and the Celo Dollar (cUSD).
```

Comment:
This is confusing: maybe remove the reference to proof of stake (should still be covered in the historical section) and reword the that "all" crypto assets are ERC-20 which is not entirely true.

Or maybe really just something along the lines of: since the migration, Celo is an OP stack based L2 on top of Ethereum, extending it by these cool features and intentions (inspiration? https://app.t2.world/article/cm1eqxyh8151217321mcesuw528v)